ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -710,8 +710,78 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-ovn
-- as: aws-ipi-ovn-hypershift-mgmt-p2-f21
-  cron: 21 21 2 * *
+- as: aws-ipi-ovn-hypershift-mce-guest-p1-f7
+  cron: 23 21 6,13,22,30 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and @hypershift-hosted and
+        not @fips'
+      MCE_VERSION: "2.3"
+      TAG_VERSION: '@4.13'
+      TEST_ADDITIONAL: Hypershift
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~NonPreRelease&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&;~Serial&;~Disruptive&
```

Comment:
@kuiwang02  Could you help check ?  I kind remember that this is not needed unless it's a disconnected or destructive tests.

Reply:
@liangxia it is added to overwrite the default value because it is hypershift hosted (guest) cluster.
there is more filter `~NonHyperShiftHOST&;` for such cluster. and this filter can not be in default value.

Reply:
Ack